yarn-project/kv-store/.mocharc.json

```diff
@@ -1,13 +1,6 @@
 {
-    "require": "ts-node/register",
-    "extensions": [
-      "ts"
-    ],
-    "spec": [
-      "./src/**/!(indexeddb)/*.test.ts"
-    ],
-    "node-option": [
-      "experimental-specifier-resolution=node",
-      "loader=ts-node/esm"
-    ]
-  }
+  "require": "ts-node/register",
+  "extensions": ["ts"],
+  "spec": ["./src/**/!(indexeddb)/*.test.ts"],
+  "node-option": ["experimental-specifier-resolution=node", "loader=ts-node/esm"]
+}
```

Comment:
From yarn format.
(A few files got modified by this, strangely).

Reply:
Sounds like we're not being too strict about said formatting or its config, given that this was passing CI before and is also passing now.